sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/jms/ServiceBusJmsConnectionFactoryConfiguration.java

```diff
@@ -62,7 +62,7 @@ public void registerBeanDefinitions(AnnotationMetadata importingClassMetadata,
             BindResult<Boolean> cacheEnabledResult = Binder.get(environment).bind("spring.jms.cache.enabled", Boolean.class);
 
             if (isPoolConnectionFactoryClassPresent()
-                && ((!cacheEnabledResult.isBound() && !poolEnabledResult.isBound())) || poolEnabledResult.orElseGet(() -> false)) {
+                && ((!cacheEnabledResult.isBound() && !poolEnabledResult.isBound()) || poolEnabledResult.orElseGet(() -> false))) {
```

Comment:
Could you explain why we are making this change?

Reply:
The unit test of the last commit will fail based on the previous version.